go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/go-piv/piv-go
 
-go 1.16
```

Comment:
These are in https://github.com/go-piv/piv-go/pull/139, and should be dropped from this PR in cleanup